aea/protocols/generator.py

```diff
@@ -184,13 +269,14 @@ def _message_class_str(self) -> str:
 
         # Imports
         cls_str += "from enum import Enum\n"
-        cls_str += "from typing import Set, Tuple, cast\n\n"
+        cls_str += (
+            "from typing import Set, Tuple, cast, Union, Dict, Optional, FrozenSet\n\n"
```

Comment:
you should only be importing those types which are actually used.

Reply:
Done